.travis.yml

```diff
@@ -1,15 +1,15 @@
 language: go
 go:
-  - 1.11.x
+  # test with the two most recent Go versions
   - 1.12.x
   - 1.13.x
 env:
-  - NSQ_DOWNLOAD=nsq-0.3.8.linux-amd64.go1.6.2 GOARCH=amd64
-  - NSQ_DOWNLOAD=nsq-0.3.8.linux-amd64.go1.6.2 GOARCH=386
   - NSQ_DOWNLOAD=nsq-1.0.0-compat.linux-amd64.go1.8 GOARCH=amd64
   - NSQ_DOWNLOAD=nsq-1.0.0-compat.linux-amd64.go1.8 GOARCH=386
   - NSQ_DOWNLOAD=nsq-1.1.0.linux-amd64.go1.10.3 GOARCH=amd64
   - NSQ_DOWNLOAD=nsq-1.1.0.linux-amd64.go1.10.3 GOARCH=386
+  - NSQ_DOWNLOAD=nsq-1.2.0.linux-amd64.go1.12.9 GOARCH=amd64
+  - NSQ_DOWNLOAD=nsq-1.2.0.linux-amd64.go1.12.9 GOARCH=386
```

Comment:
I just had an idea for reducing the test matrix here - only test `GOARCH=386` with one version of nsqd/nsqlookupd. We'll still have GOARCH=386 tested with all versions of go, just not with all versions of nsqd ...